pkgs/servers/scylladb/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv,
+  fetchgit,
+  python3Packages,
+  python3,
+  pkgconfig,
+  gcc8Stdenv,
+  boost,
+  git,
+  systemd,
+  gnutls,
+  cmake,
+  makeWrapper,
+  ninja,
+  ragel,
+  hwloc,
+  jsoncpp,
+  libantlr3cpp,
+  antlr3,
+  numactl,
+  protobuf,
+  cryptopp,
+  libxfs,
+  libyamlcpp,
+  libsystemtap,
+  lksctp-tools,
+  lz4,
+  libxml2,
+  zlib,
+  libpciaccess,
+  snappy,
+  libtool,
+  thrift-0_10
+}:
+gcc8Stdenv.mkDerivation rec {
+  pname = "scylladb";
+  version = "3.0.5";
+
+  src = fetchgit {
+    url = "https://github.com/scylladb/scylla.git";
+    rev = "403f66ecad6bc773712c69c4a80ebd172eb48b13";
+    sha256 = "14mg0kzpkrxvwqyiy19ndy4rsc7s5gnv2gwd3xdwm1lx1ln8ywsi";
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./seastar-configure-script-paths.patch ];
+
+  nativeBuildInputs = [
+   pkgconfig
+   cmake
+   makeWrapper
+   ninja
+  ];
+
+  buildInputs = [
+   python3Packages.pyparsing
+   python3
+   boost
+   git
+   systemd
+   gnutls
+   ragel
+   hwloc
+   jsoncpp
+   libantlr3cpp
+   numactl
+   antlr3
+   protobuf
+   cryptopp
+   libxfs
+   libyamlcpp
+   libsystemtap
+   lksctp-tools
+   lz4
+   libxml2
+   zlib
+   libpciaccess
+   snappy
+   libtool
+   thrift-0_10
+  ];
+
+  configurePhase = ''
+     patchShebangs ./configure.py
```

Comment:
Maybe move this to the `prePatchPhase` or at least `preConfigure`. The main point being don't override `configurePhase` as that can cause problems and skip steps.

Reply:
Thanks! Turns out I don't need this at all ^__^ removed it.

Reply:
Turns out `patchShebangs` *is* needed, I just happened to have the binary on my system and no sandboxing. Moved this into the `postPatch` phase as you recommended.